snuba/web/views.py

```diff
@@ -97,10 +98,12 @@ def check_clickhouse() -> bool:
 application.url_map.converters["dataset"] = DatasetConverter
 
 
-@application.errorhandler(BadRequest)
-def handle_bad_request(exception: BadRequest):
+@application.errorhandler(InvalidJsonRequestException)
+def handle_invalid_json(
+    exception: InvalidJsonRequestException,
+) -> Tuple[str, int, Mapping[str, str]]:
```

Comment:
It might make sense to type alias this return type (or I think we can use some sort of `Response` object, but it's been a while since I've worked in this file.)